FrostyEditor/Views/Windows/ProfileSelectWindow.axaml

```diff
@@ -3,22 +3,72 @@
         xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
         xmlns:vm="clr-namespace:FrostyEditor.ViewModels.Windows"
-        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+        mc:Ignorable="d"
+        Width="1000" Height="600"
         x:Class="FrostyEditor.Views.Windows.ProfileSelectWindow"
-        Title="ProfileSelectWindow"
+        Title="{StaticResource ID_PROFILE_TITLE}"
         x:DataType="vm:ProfileSelectWindowViewModel"
-        x:CompileBindings="True">
+        x:CompileBindings="True"
+	    UseLayoutRounding="True" RenderOptions.BitmapInterpolationMode="HighQuality"
+        WindowState="Normal" WindowStartupLocation="CenterScreen"
+        Background="{x:Null}"
+        TransparencyLevelHint="AcrylicBlur"
+        Foreground="{DynamicResource DockThemeForegroundBrush}"
+        BorderThickness="1" BorderBrush="{DynamicResource DockThemeBorderLowBrush}"
+        ExtendClientAreaToDecorationsHint="True"
+        ExtendClientAreaChromeHints="PreferSystemChrome"
+	    ExtendClientAreaTitleBarHeightHint="-1"
+		CanResize="False">
+
     <Design.DataContext>
         <vm:ProfileSelectWindowViewModel />
     </Design.DataContext>
-    <Grid Background="Transparent">
+	
+    <Grid Background="Transparent" Margin="2">
         <Grid.RowDefinitions>
+            <RowDefinition Height="50"/>
             <RowDefinition Height="*"/>
             <RowDefinition Height="Auto"/>
         </Grid.RowDefinitions>
-        
-        <Grid Grid.Row="0">
-            <ListBox ItemsSource="{Binding Profiles}" SelectedItem="{Binding SelectedProfile}">
+
+		<ExperimentalAcrylicBorder IsHitTestVisible="False">
```

Comment:
First of all join the frosty discord and head to the #developers channel, since its probably better to discuss what kind of theme frosty should use. Second things like this should go into its own control, sth like FrostyWindow which has the icon at the top and a name prop, etc so it doesnt need to duplicated for every window

Reply:
Try to share the server instead of copying from your browser.

Reply:
updated it

Reply:
> updated it

Omg I'm already in that channel but I don't see #developers, maybe i dont have permission

![Screenshot in #read-me](https://github.com/CadeEvs/FrostyToolsuite/assets/100991210/7abdd4d3-ec2b-4ba9-8192-1fbc2a02e6d8)
maybe this channel? translate: 🔒No access